indico/modules/events/abstracts/templates/abstract/timeline.html

```diff
@@ -96,10 +103,11 @@
                     {% endif %}
                 </div>
                 <div>
-                    {{ render_track(review.track, classes='abstract-track-tag i-tag outline') }}
+                    {{ render_track(review.track) }}
                 </div>
             </div>
-            <div class="i-timeline-item-box header-indicator-top">
+            <div class="i-timeline-item-box header-indicator-top {% if not review.comment %}header-only{% endif %}"
+                 {% if not review.comment %}data-no-comment{% endif %}>
```

Comment:
I think it was actually you starting it, I always used the camelCase versions :P